changelog.d/100.fixed

```diff
@@ -0,0 +1,2 @@
+Explicitly list `packaging` as a dependency so `pipx` installs it in the virtual environment instead of relying on
+`setuptools` to pull it in.
```

Comment:
According to [the documentation](https://towncrier.readthedocs.io/en/stable/tutorial.html#creating-news-fragments) it's `bugfix`, not `fixed`. (unless it accepts both? But still probably best to be standard)

Also I think you might be able to give this a `.md` suffix to have it interpreted as Markdown. Like `100.bugfix.md`.

Reply:
Yep, no problems… I thought it was just `fixed`, but I'll rename. :-)